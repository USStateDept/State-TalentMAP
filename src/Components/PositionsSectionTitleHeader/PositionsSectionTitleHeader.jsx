import PropTypes from 'prop-types';

const PositionsSectionTitleHeader = ({ title }) => (
  <div className="positions-section-title-header">
    <span>
      {title}
    </span>
  </div>
);

PositionsSectionTitleHeader.propTypes = {
  title: PropTypes.node.isRequired,
};

export default PositionsSectionTitleHeader;
