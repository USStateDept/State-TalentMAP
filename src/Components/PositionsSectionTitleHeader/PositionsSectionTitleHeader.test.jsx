import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import PositionsSectionTitleHeader from './PositionsSectionTitleHeader';

describe('PositionsSectionTitleHeaderComponent', () => {
  it('is defined', () => {
    const wrapper = shallow(
      <PositionsSectionTitleHeader
        title="title"
      />,
    );
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(
      <PositionsSectionTitleHeader
        title="title"
      />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
