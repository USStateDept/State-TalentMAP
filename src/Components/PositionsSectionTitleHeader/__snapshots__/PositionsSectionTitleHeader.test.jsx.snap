// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PositionsSectionTitleHeaderComponent matches snapshot 1`] = `
<div
  className="positions-section-title-header"
>
  <span>
    title
  </span>
</div>
`;
